pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -585,7 +602,12 @@ func TestScaleBounds(t *testing.T) {
 			}
 			pa.Spec.Reachability = tc.reachability
 
-			min, max := pa.ScaleBounds()
+			config := tc.config
+			if config == nil {
+				config, _ = autoscalerconfig.NewConfigFromMap(map[string]string{})
```

Comment:
do we need this or could we just use an empty `autoscaler.Config` for the other cases? (that way we can avoid the nil check and just use the zero value in the table). I'd think since the default of DefaultMaxScale is zero (unlimited) and that's the zero value anyway that might work?

Reply:
yup, this worked. much cleaner. thanks for the pointer!